src/Features/CSharp/Portable/AddImport/CSharpAddImportFeatureService.cs

```diff
@@ -374,8 +375,9 @@ private string GetUsingDirectiveString(INamespaceOrTypeSymbol namespaceOrTypeSym
                 }
 
                 var addImportService = document.GetLanguageService<IAddImportsService>();
+                var generator = SyntaxGenerator.GetGenerator(document);
                 var newRoot = addImportService.AddImports(
-                    semanticModel.Compilation, root, contextNode, newImports, placeSystemNamespaceFirst, cancellationToken);
+                    semanticModel.Compilation, root, contextNode, newImports, generator, placeSystemNamespaceFirst, cancellationToken);
```

Comment:
When C# and VB SyntaxGenerator singleton instances are available in the shared layer, we can clean this up to avoid explicitly passing down the generator into add imports service .

Reply:
Sounds good.  I can do that in my PR, or right after.